packages/stg-evm-usdt-v2/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "tether-contracts-hardhat",
+  "version": "1.0.0",
+  "description": "",
+  "keywords": [],
+  "license": "ISC",
+  "author": "",
+  "main": "index.js",
+  "scripts": {
+    "compile": "hardhat compile",
+    "test": "hardhat test"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "4.2.0",
+    "@openzeppelin/contracts-upgradeable": "4.2.0"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
+    "@nomicfoundation/hardhat-toolbox": "^2.0.2",
+    "@nomicfoundation/hardhat-verify": "^1.0.1",
+    "@nomiclabs/hardhat-etherscan": "^3.1.7",
+    "@openzeppelin/hardhat-upgrades": "^1.27.0",
+    "chai": "^4.3.7",
+    "hardhat": "^2.15.0",
```

Comment:
Pls check that the versions of `hardhat` and `chai` and these helper libraries match the other `package.json` files in this repo - to prevent having multiple versions of hardhat around

Reply:
Ah great call!